packages/wc-compass/src/layouts/about/_about.scss

```diff
@@ -0,0 +1,81 @@
+div.cdg-about-section {
+  display: flex;
+  flex-direction: row;
+  color: var(--grey100);
+  padding-top: 32px;
+  padding-bottom: 32px;
+  container-name: CdgAbout;
+  container-type: inline-size;
+  justify-content: space-between;
+
+  .cdg-about-left {
+    width: 50%;
+  }
+
+  .cdg-about-right {
+    background: var(--accent);
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+}
+
+@media screen and (max-width: 960px) {
+  div.cdg-about-section {
+    flex-direction: column;
+  }
+
+  .cdg-about-left {
+    width: 100% !important;
+    overflow: hidden;
+  }
+}
+
+.cdg-about-right {
+  display: flex;
+  flex-direction: column;
+  color: white;
+
+  .cdg-about-header {
+    font-size: 3rem;
+    line-height: 3.5rem;
+    margin: 0;
+  }
+
+  &-wrap {
```

Comment:
Please use fully class name instead of this style. For example if you are inspecting an element with class name .cdg-about-header-wrap on screen. It's so hard to find where is the css to fix.

Reply:
fixed